var/spack/repos/builtin/packages/netcdf-fortran/package.py

```diff
@@ -33,8 +33,13 @@ class NetcdfFortran(AutotoolsPackage):
     variant("doc", default=False, description="Enable building docs")
 
     depends_on("netcdf-c")
+    depends_on("netcdf-c+shared", when="+shared")
     depends_on("netcdf-c@4.7.4:", when="@4.5.3:")  # nc_def_var_szip required
     depends_on("doxygen", when="+doc", type="build")
+    depends_on("mpi", when="^netcdf-c+parallel-netcdf")
```

Comment:
Lines 39-42 should not be needed. netcdf-c depends on MPI and hdf5, that should be sufficient? We've been building parallel netcdf libraries with Fortran API so far, and didn't have any problems.